### ISSUE TITLE
RenderTargetBitmap take screenshot at wrong place

### ISSUE THREAD

**Issue by iurycarlos** [unoplatform/uno#16836]
### Current behavior

RenderTargetBitmap RenderAsync starts the screenshot from the 0,0 point of the element's parent, instead considering the element's top left.

Consider a page, with a Stackpanel.
Inside of the SP you have only a Button.

if we use a RenderTargetBitmap to take a screenshot from the button like:

        var targetBitmap = new RenderTargetBitmap();
                        
        await targetBitmap.RenderAsync(Button);

It will generate a correct targetBitmap perfectly.

However, if you add another child to the StackPanel, but inserted before the Button, that will generate a blank bitmap, depending on the width of the new child.

Basically, RenderAsync is not considering the position of the element inside its parent. The result is that the screenshot only consider the element visual, but the copy start at the 0,0 point in the element's parent.

### Expected behavior

Should consider the position of element, not its parent.

### How to reproduce it (as minimally and precisely as possible)

```


    public static async Task<FrameworkElement> GenerateBitmapAsync(FrameworkElement element)
    {
        //adding those lines will generate the bitmap correctly, however it will reposition the element

        //element.Measure(new Windows.Foundation.Size(double.PositiveInfinity, double.PositiveInfinity));
        //element.Arrange(new Windows.Foundation.Rect(0, 0, element.DesiredSize.Width, element.DesiredSize.Height));

        var targetBitmap = new RenderTargetBitmap();
                        
        await targetBitmap.RenderAsync(element);

        var pixels = await targetBitmap.GetPixelsAsync();
        var stream = new InMemoryRandomAccessStream();

        var borderBrush = new SolidColorBrush(Colors.Red);
        if (pixels.ToArray().Any(p => p > 0))
        {
            borderBrush = new SolidColorBrush(Colors.Green);
        }

        var encoder = await BitmapEncoder.CreateAsync(BitmapEncoder.PngEncoderId, stream);

        encoder.SetPixelData(
            BitmapPixelFormat.Bgra8,
            BitmapAlphaMode.Premultiplied,
            (uint)targetBitmap.PixelWidth,
            (uint)targetBitmap.PixelHeight,
            96,
            96,
            //DisplayInformation.GetForCurrentView().RawDpiX,
            //DisplayInformation.GetForCurrentView().RawDpiY,
            pixels.ToArray());

        await encoder.FlushAsync().AsTask();

        var image = new Image
        {
            Source = targetBitmap,
            Width = element.ActualWidth,
            Height = element.ActualHeight,
            IsHitTestVisible = false,
        };

        var border = new Border
        {
            Child = image,
            BorderBrush = borderBrush,
            BorderThickness = new Thickness(2),
        };

        return border;
    }
 
    private async void Test_Click(object sender, RoutedEventArgs e)
    {
        var result = await GenerateBitmapAsync(TestButton);

        if (result is UIElement uiEl)
        {
            SPTest.Children.Add(uiEl);
        }
    }


```
```

		<StackPanel
			x:Name="SPTest"
			Grid.Row="1"
			HorizontalAlignment="Center"
			VerticalAlignment="Center"
			Spacing="16">

			<TextBlock
				Height="50"
				Text="I'm here to make troubles."
				Visibility="Visible" />

			<Button
				x:Name="TestButton"
				Height="100"
				Click="Test_Click"
				Content="Test Render" />
			<TextBlock Text="Here i don't make troubles" />

		</StackPanel>


```

Used:
(Uno.Templates::5.2.37)

dotnet new unoapp -o TestScreenshotRender2 -preset "recommended" -platforms "android" -platforms "windows" -platforms "desktop" -platforms "wasm" -theme-service  -vscode False -pwa False -theme "fluent" -presentation "mvvm" -config  -di  -log "default" -nav "regions" -http  -loc  -tests "none" -toolkit  -dsp False -id "com.companyname.TestScreenshotRender2" -pub "O=TestScreenshotRender2"
The template "Uno Platform App" was created successfully.

### Works on UWP/WinUI

None

**Comment by Youssef1313** [comment]
This doesn't repro in latest devs.

**Comment by Youssef1313** [comment]
The PR that likely fixed this is #15875
